changelogs/fragments/832-acme-challenges.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
```

Comment:
arguably, being lenient with non-compliant CAs is a new feature of this module, rather than a bug fix, but that's hair-splitting

Reply:
True :)

Reply:
```suggestion
minor_changes:
```